codechecker_lib/analyzers/result_handler_base.py

```diff
@@ -183,17 +183,24 @@ def get_analyzer_result_file(self):
                 '_' + analyzed_file_name + '_' + uid + '.plist'
 
             out_file = os.path.join(self.__workspace, out_file_name)
-            self.__res_file = out_file
+            self.__result_file = out_file
+
+        return self.__result_file
 
-        return self.__res_file
+    @analyzer_result_file.setter
```

Comment:
It was introduced in order to be able to reduce the code duplication between plist processing and checking. It might be moved to a constructor argument, but in that case several methods and functions will need an extra argument.